Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -276,16 +294,28 @@ ReshapableInput::ReshapeData ReshapableInput::Reshape(ControlState x, ControlSta
   y -= m_center.y;
 
   // TODO: make the AtAngle functions work with negative angles:
-  const ControlState angle = std::atan2(y, x) + MathUtil::TAU;
+  ControlState angle = std::atan2(y, x) + MathUtil::TAU;
 
-  const ControlState gate_max_dist = GetGateRadiusAtAngle(angle);
   const ControlState input_max_dist = GetInputRadiusAtAngle(angle);
+  ControlState gate_max_dist = GetGateRadiusAtAngle(angle);
 
   // If input radius (from calibration) is zero apply no scaling to prevent division by zero.
   const ControlState max_dist = input_max_dist ? input_max_dist : gate_max_dist;
-
+  
   ControlState dist = Common::DVec2{x, y}.Length() / max_dist;
 
+  const double virtual_notch_size = GetVirtualNotchSize();
+
+  if (virtual_notch_size > 0.0 && dist >= MINIMUM_NOTCH_DISTANCE)
```

Comment:
Move this whole if-block to before where `gate_max_dist` is first defined (line 300). Re-assigning it here won't fix all the resulting distance calculations.

Reply:
The problem I have is that `gate_max_dist` is used to calculate the value of `dist` which means I can't just put this block beforehand since `dist` is now used to check against `MINIMUM_NOTCH_DISTANCE`.

Reply:
Ah sorry. I missed the `dist` bit. :P